### PR TITLE
AUTO_INCREMENT: MySQL compatibility mode GA

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

Since TiDB v6.5.0, the MySQL compatibility mode of AUTO_INCREMENT is generally available.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.5 (TiDB 6.4 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by tiancaiamao — APPROVED** [review]

File: auto-increment.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ aliases: ['/docs-cn/dev/auto-increment/']
 
 > **注意：**
 >
-> 如果要求自增编号在所有 TiDB 实例上具有单调性，并且你的 TiDB 版本在 v6.4.0 及以上，你可以使用 v6.4.0 引入的实验特性 [MySQL 兼容模式](#mysql-兼容模式)。
+> 如果要求自增编号在所有 TiDB 实例上具有单调性，并且你的 TiDB 版本在 v6.4.0 及以上，推荐使用 [MySQL 兼容模式](#mysql-兼容模式)。
```
Comment (qiancai) [inline auto-increment.md:23]: "v6.4.0 及以上" 是否需要改为 "v6.5.0 及以上"，因为 v6.4.0 上还是实验特性

Comment (Oreoxmt) [inline auto-increment.md:23]:
```suggestion
> 如果要求自增编号在所有 TiDB 实例上具有单调性，并且你的 TiDB 版本在 v6.4.0 及以上，你可以使用 [MySQL 兼容模式](#mysql-兼容模式)。
>
> - MySQL 兼容模式在 TiDB v6.4.0 上是实验特性，不推荐在生产环境中使用。
> - MySQL 兼容模自 TiDB v6.5.0 起成为正式功能 (GA)，可用于生产环境。
```

Comment (Oreoxmt) [reply auto-increment.md:23]:
```suggestion
> 如果要求自增编号在所有 TiDB 实例上具有单调性，并且你的 TiDB 版本在 v6.5.0 及以上，推荐使用 [MySQL 兼容模式](#mysql-兼容模式)。
```

Comment (tiancaiamao) [reply auto-increment.md:23]: OK

**Review by qiancai — APPROVED** [review]

**Pull request merged**